changelog.d/7053.feature

```diff
@@ -0,0 +1 @@
+Add options to disable setting profile info for prevent changes.
```

Comment:
Sorry, but I'm not sure to understand what you mean here by "for prevent changes".

FWIW I would phrase this changelog entry like this: "Add options to prevent users from changing their profile or associated 3PIDs."